### DISCUSSION TITLE
How to stop code-server

### DISCUSSION THREAD

**Discussion by ctoabidmaqbool** [coder/code-server#6065] [Q&A] [answered]
I have installed code-server using this method on Ubuntu:

```
curl -fsSL https://code-server.dev/install.sh | sh
```

I can simple code-server using:
```
code-server
```

How can I stop code-server completely?

**Comment by DXShelley** [comment]
netstat -ap | grep 1200
kill xxx

**Comment by code-asher** [comment, marked as answer]
If you are running plain `code-server` then you could `Ctrl+c` the command.  If you run it in the background manually like with `code-server &` then you could kill it by finding the PID via `ps` or `netstat` and run `kill <pid>` or match by name like with `pkill code-server`.

If you can, run code-server with systemd (`systemctl start code-server`).  Then you can let systemd manage the lifecycle and you can run `systemctl stop code-server`.

**Reply by ctoabidmaqbool** [reply]
`systemctl stop code-server` works in my case! Thanks for providing solution.
